src/pages/PrintPage.tsx

```diff
@@ -163,6 +163,14 @@ class SummaryPage extends React.Component<RouteComponentProps, State> {
       bmdConfig,
     } = this.context.election
     const { showHelpPage, showSettingsPage } = bmdConfig
+
+    const contests: Array<CandidateContest | YesNoContest> = this.context
+      .contests
+
+    if (!contests.length) {
+      return <Redirect to="/reset" />
+    }
+
```

Comment:
If the only way to this page is by going through some contests… wouldn't `contests` always have a length?